NoMansLand/Assets/Scripts/Player/HandleObject.cs

```diff
@@ -36,7 +34,7 @@ void Update()
             if (spaceship != null && 
                 Vector3.Distance(child.gameObject.transform.position, spaceship.transform.position) <= DISTANCE_LIMIT)
             {
-                SpaceshipTimekeeping.AddBonusTime(TIME_BONUS);
+                SpaceshipManager.AddPartToShip();
```

Comment:
We are not using `HandleObject.cs` at all anymore since it's non-controller. Should we move this code (or duplicate this code) to `PlayerInteractions` so it's compatible with controller?

Reply:
done